Backend/app/routers/login.py

```diff
@@ -1,12 +1,12 @@
 import json
-from datetime import datetime, timedelta, timezone
+from datetime import UTC, datetime, timedelta
```

Comment:
I made the exact same change some weeks ago and the app broke. In the future i will check what happened but it should be revert for now.

Reply:
Hey, i think this was not reverted. Once its reverted i will run the tests and merge :)